### ISSUE TITLE
compiler error in anaconda qutip

### ISSUE THREAD

**Issue by caseyfitz** [qutip/qutip#409]
[Terminal Saved Output.txt](https://github.com/qutip/qutip/files/67295/Terminal.Saved.Output.txt)
Hi, I hope I'm handling this correctly, as I'm new to both qutip and git (recent MATLAB deserter)...

I've been trying to set up qutip within my anaconda distribution using 

    conda install -c https://conda.anaconda.org/jrjohansson qutip

The package linking completes with no errors. But when I 

    import qutip

I get the following

    ImportError: Building module qutip.cy.spmatfuncs failed: 
    ["CompileError: command 'gcc' failed with exit status 1\n"]

After running
    
    conda update gcc
    conda update cython

I get the same error.

I am running OS X 10.11.1.

I'm really looking forward to using this software for thesis work, so I hope to get resolved ASAP! Thanks for reading.

**Comment by caseyfitz** [comment]
RESOLVED.

First, I changed to a source install inside the anaconda site-packages folder instead of pulling from the site. I was still getting the error, but I found this similar issue, which suggested I remove the fpmath calls in a couple of the c compiler setup.py files. In particular, the setups in the cy and control subdirectories (see links). For instance, something like 

            extra_compile_args=['-w', '-ffast-math',
                                '-O3', '-march=native, '-mfpmath=sse']


to

            extra_compile_args=['-w', '-ffast-math',
                                '-O3', '-march=native'], #, '-mfpmath=sse'


For some reason, this wasn't working either, so I decided to try changing
    -march=native

to
     -march=x86-64

, and that did the trick!
